packages/bee-q/src/components/toast/scss/bq-toast.variables.scss

```
@@ -0,0 +1,42 @@
+/* -------------------------------------------------------------------------- */
+/*                          Toast custom properties           */
+/* -------------------------------------------------------------------------- */
+
+:host {
+  /**
+   * @prop --bq-toast--background: Toast background color
+   * @prop --bq-toast--box-shadow: Toast box shadow
+   * @prop --bq-toast--padding-y: Toast vertical padding
+   * @prop --bq-toast--padding-x: Toast horizontal padding
+   * @prop --bq-toast--gap: Toast distance between icon and text
+   * @prop --bq-toast--border--radius: Toast border radius
```

I think it should be:

```suggestion
   * @prop --bq-toast--border-radius: Toast border radius
```

---

Done